openmc/data/thermal.py

```diff
@@ -588,7 +588,8 @@ def from_ace(cls, ace_or_filename, name=None):
         return table
 
     @classmethod
-    def from_njoy(cls, filename, filename_thermal, temperatures=None, **kwargs):
+    def from_njoy(cls, filename, filename_thermal, temperatures=None,
+                  error=0.001, **kwargs):
```

Comment:
Same thing here -- move this argument to `njoy.make_ace_thermal` and let `**kwargs` pass it if needed.